packages/web-client/app/components/workflow-thread/index.ts

```diff
@@ -40,14 +47,15 @@ export default class WorkflowThread extends Component<WorkflowThreadArgs> {
       // if prefers-reduced-motion, don't autoscroll, and release postables as soon as available
       // this puts control on when and how to scroll in users' hands
       this.autoscroll = false;
-      this.workflow.interval = 0;
-      this.threadAnimationInterval = `0ms`;
+      this.threadAnimationInterval = 0;
+      this.cssThreadAnimationInterval = '0ms';
```

Comment:
`cssThreadAnimationInterval` seems like it can become a getter that tacks on `"ms"` to `threadAnimationInterval`, or even be removed in favour of simply concatenating within the template?